### DISCUSSION TITLE
为啥所有的源速度都很慢呀，用的jellyfin去读取的

### DISCUSSION THREAD

**Discussion by scientificCommunity** [iptv-org/community#75] [Q&A] [answered]
为啥所有的源速度都很慢呀，用的jellyfin去读取的

Translation : Why are the channels so slow ? I use Jellyfish to watch.

**Comment by BellezaEmporium** [comment, marked as answer]
It might be due to multiple reasons.

1) Server bandwidth is low
2) Your bandwidth is low
3) Server instability (bigger chance if the source is a self-hosted server)

Depending on the channel, you'll have different reasons about why it is slow.

If you're onto Chinese channels only, 60% of them are mostly China Telecom TV/ZTE Telecom restreams. I'm almost sure those are self-hosted (or badly secured ISP streams)
